### PR TITLE
compiler and interpreter resources

### PR DESCRIPTION
**Pull request by king-11**
- writing a c compiler
- crafting interpreters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,11 +12,13 @@ A curated list of awesome programming language design and implementation resourc
 **Resources for general learning about compilers**
 - [Stanford Online - Compilers](https://online.stanford.edu/courses/soe-ycscs1-compilers) - Discusses the major ideas used today in the implementation of programming language compilers
 - [Writing an Compiler in Go by Thorsten Ball](https://compilerbook.com/) - eBook covering the step-by-step implementation of a compiler using Go
+- [Writing a C Compiler](https://norasandler.com/2017/11/29/Write-a-Compiler.html) - a language agnostic series on writing your own C compiler
```
Comment (savannahostrowski) [inline README.md]:
```suggestion
- [Writing a C Compiler](https://norasandler.com/2017/11/29/Write-a-Compiler.html) - A language agnostic series on writing your own C compiler
```

File: README.md [hunk 2]
```diff
@@ -12,11 +12,13 @@ A curated list of awesome programming language design and implementation resourc
...
 
 
 ## Interpreters
 **Resources for general learning about interpreters**
 - [Writing an Interpreter in Go by Thorsten Ball](https://interpreterbook.com/) - eBook covering the step-by-step implementation of an interpreter using Go
+- [Crafting Interpreters](https://craftinginterpreters.com/) - book covering implementation of a full-featured, efficient scripting language
```
Comment (savannahostrowski) [inline README.md]:
```suggestion
- [Crafting Interpreters](https://craftinginterpreters.com/) - A book covering implementation of a full-featured, efficient scripting language
```

**Review by savannahostrowski — CHANGES_REQUESTED** [review]
Nits but LGTM

**Comment by king-11** [comment]
Made the requested changes

**Review by savannahostrowski — APPROVED** [review]

**Comment by savannahostrowski** [comment]
Thanks! :)

**Pull request merged**